types/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const supportedChains = [
```

Comment:
Supported chains should be requested from the server

Reply:
You mean this should be a query that gets all chain slugs from the database? I thought about implementing that way, but that would be an extra query thatt would have to be done every time some address is input. Having a constant that corresponds to the supported chains in the database (this correspondence will be enforced by unittests) looked like a more efficient solution.